### ISSUE TITLE
ERROR_METHOD_NOT_FOUND in bx24.call_batch

### ISSUE THREAD

**Issue by maglor** [yarbshk/bitrix24-python3-client#1]
Hello! Batch call from example gives error:

```
>>> bx24.call_batch({
...    'get_user': ['user.current', {}],
...    'get_department': {
...        'method': 'department.get',
...        'params': {'ID': '$result[get_user][UF_DEPARTMENT]'}
...    }
... })
{'result': {'result': [], 'result_error': {'get_user': {'error': 'ERROR_METHOD_NOT_FOUND', 'error_description': 'Method not found!'}, 'get_department': {'error': 'ERROR_METHOD_NOT_FOUND', 'error_description': 'Method not found!'}}, 'result_total': [], 'result_next': []}}
```

**Comment by yarbshk** [comment]
Hi @maglor! Yes, it is! The `call_batch` method doesn't work properly now (unfortunatelly). I think it's probably a Bitrix24-side bug (referensing to [this](https://forums.bitrix24.ru/forum1/topic3965/) old topic). To make it clear I sent a bug request into the support center of Bitrix24 a few days ago (tech-supports of the center reply so slow for owners of a free account). We are trying to resolve this issue now. Follow this issue to keep in touch pro tempore. Thanks!

**Comment by maglor** [comment]
@yarbshk Thanks! 
I'm B24 partner, I'll try to ask support about this bug too, if you provide a ticket number. May be they will reply a little bit faster :-)

**Comment by yarbshk** [comment]
@maglor nice to hear you! Yep, sounds good) The ticket number is №1616327. Hope we resolve this issue a little bit faster. Thanks for indifference! Where can I follow about a work progress? How could I help?

**Comment by maglor** [comment]
> Там вызов делается не к batch, а без указания этого метода, поэтому и проблема.
> в примере не указан ключ auth с токеном авторизации.
>
> Корректный вызов:
> `/rest/batch.json?cmd[get_user]=user.current?&cmd[get_department]=department.get?ID=%24result%5Bget_user%5D%5BUF_DEPARTMENT%5D`

**Comment by yarbshk** [comment]
@maglor thanks, but it's not a reason of the broken `call_method` method. The structure of API call was  not quite right. [This article](https://dev.1c-bitrix.ru/rest_help/general/batch.php) give me better understanding of the topic. Fix this issue in the 3b22beab47c98b216966a384768b7124eae6faad  commit. Thanks a lot for your contribution! Close this issue.
